lib/table_rex/renderer/text.ex

```diff
@@ -128,9 +129,10 @@ defmodule TableRex.Renderer.Text do
   defp render_header({%Table{header_row: header_row} = table, meta, opts, rendered}) do
     row_height = Meta.row_height(meta, 0)
     separator = if meta.render_column_separators?, do: opts[:vertical_symbol], else: " "
-    line = render_cell_row(table, meta, header_row, separator)
-    if meta.render_vertical_frame? do
-      line = line |> frame_with(opts[:vertical_symbol])
+    line = if meta.render_vertical_frame? do
+      render_cell_row(table, meta, header_row, separator) |> frame_with(opts[:vertical_symbol])
+    else
+      render_cell_row(table, meta, header_row, separator)
     end
```

Comment:
Same with this block 132-135. The common action can be done before the branch to make it clearer what the branch is for.